sycl/test/lit.cfg.py

```diff
@@ -76,6 +76,7 @@
 config.substitutions.append( ('%sycl_source_dir', config.sycl_source_dir) )
 config.substitutions.append( ('%opencl_include_dir',  config.opencl_include_dir) )
 config.substitutions.append( ('%cuda_toolkit_include',  config.cuda_toolkit_include) )
+config.substitutions.append( ('%cuda_plugin_dir',  config.cuda_plugin_dir) )
```

Comment:
Why would you need this?

Reply:
backend/cuda/primary_context.cpp tests directly uses pi_cuda.hpp which is inside sycl/plugins/cuda which is out of default include search path

Reply:
I think `backend/cuda/primary_context.cpp` should be using GoogleTest infrastructure instead of LIT.